migrations/20220110135403_add-farm-onboarding-extension.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function (knex) {
+  return knex.raw(`
+    INSERT INTO extensions(type, created_at) VALUES ('farm-onboarding', current_timestamp)
```

Comment:
extensions is still a bit confusing of a name to me, can we call it group_extensions at least?

Reply:
group_extensions is really close to the join table's name, groups_extensions. This is the same pattern as widgets and groups_widgets.